src/styles/blocks/header.scss

```diff
@@ -0,0 +1,55 @@
+.header {
+  display: flex;
+  gap: 30px;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  font-weight: 500;
+
+  &__logo {
+    height: 40px;
+    width: 40px;
+  }
+}
+
+.nav {
```

Comment:
that's two separate bem blocks, move one of them to another file, 9'th point in checklist